content/probability/README.md

```diff
@@ -3,6 +3,9 @@
 Incharge: Kiran Vasudev
```

Comment:
Not sure if this belongs to your merge request, maybe @kiranvasudev can comment a bit?